README.md

```diff
@@ -46,7 +46,7 @@ Save can be used not only with static analyzers, but can be used as a test frame
 
 ## How to start
 1. Prepare and configure your test base in the proper format. See [test_detection](#test_detection) and [plugins](#plugins)
-2. Run the following: TBD (FixMe)
+2. Run the following: save --testConfig = "/my/path/to-test/save.toml"
```

Comment:
Do we support arguments syntax with `=`? And I think there are usually no spaces around `=` in CLI programs. kotlinx.cli has some configuration for `=` or ` ` syntax, not sure which one we have chosen

Reply:
yeah of course, we don't support, this is awkward